sdcm/cluster_k8s/eks.py

```diff
@@ -164,6 +167,8 @@ def __init__(
             is_deployed: bool = False,
             user_data: str = None,
     ):
+        arch = get_arch_from_instance_type(instance_type)
+        image_type = ARCH_TO_IMAGE_TYPE_MAPPING.get(arch)
```

Comment:
Probably we should raise explicit error in case the `arch` is absent in the `ARCH_TO_IMAGE_TYPE_MAPPING`.
For now it will continue with the `None` value in this case.